packages/core/src/configuration.ts

```diff
@@ -166,7 +166,8 @@ export const bars = {
 	spacing: {
 		bars: 0.2,
 		datasets: 0.25
-	}
+	},
+	maxWidth: -1
```

Comment:
Why -1? Why not `null`?

Reply:
Just needed an empty value, we can use `null` for sure 👍